src/main/java/com/twilio/http/NetworkHttpClient.java

```diff
@@ -140,7 +140,7 @@ public <T extends IRequest> Response makeRequest(final T request) {
             }
         }
 
-        if (method == HttpMethod.POST) {
+        if (method == HttpMethod.POST || method == HttpMethod.PUT) {
```

Comment:
GET does not have body all other methods can have body.
Check with Shubham before adding Delete support.

Reply:
checked with Shubham
Adding support for all methods except GET